src/App.js

```diff
@@ -1,14 +1,54 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import styled from "styled-components/macro";
+import { loadFromLocal, saveToLocal } from "./utils/localStorage";
 import PlacesPage from "./pages/PlacesPage/PlacesPage";
-/*import { Route, Switch } from "react-router-dom";*/
+import Home from "./pages/HomePage/Home";
+import locations from "./data.json";
+import { Route, Switch } from "react-router-dom";
+import Navbar from "./components/Navbar/Navbar";
+import FavoritesPage from "./pages/FavoritesPage/FavoritesPage";
 
 export default function App() {
+  const [places, setPlaces] = useState(loadFromLocal("places") ?? locations);
+  const favorites = places.filter((place) => place.isFavorite);
+
+  useEffect(() => {
+    saveToLocal("places", locations);
```

Comment:
```suggestion
    saveToLocal("places", places);
```

doesn't this have to be places? otherwise you are just saving your default data in local storage everytime places changes